dora/shell/src/main/java/alluxio/cli/fs/command/StopSyncCommand.java

```diff
@@ -1,73 +0,0 @@
-/*
- * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
- * (the "License"). You may not use this work except in compliance with the License, which is
- * available at www.apache.org/licenses/LICENSE-2.0
- *
- * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
- * either express or implied, as more fully set forth in the License.
- *
- * See the NOTICE file distributed with this work for information regarding copyright ownership.
- */
-
-package alluxio.cli.fs.command;
-
-import alluxio.AlluxioURI;
-import alluxio.annotation.PublicApi;
-import alluxio.cli.CommandUtils;
-import alluxio.client.file.FileSystemContext;
-import alluxio.exception.AlluxioException;
-import alluxio.exception.status.InvalidArgumentException;
-
-import org.apache.commons.cli.CommandLine;
-
-import java.io.IOException;
-
-/**
- * This class represents a stop sync command.
- */
-@PublicApi
-public class StopSyncCommand extends AbstractFileSystemCommand {
```

Comment:
There's a StartSyncCommand too. `StopSyncCommand` and `StartSyncCommand` should be removed in pair.

Reply:
done